jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -1173,7 +1173,7 @@ public VarHandle resolveConstantDesc(MethodHandles.Lookup lookup) throws Reflect
 					}
 					break;
 				default:
-					/*[MSG "K0619", "Malformated VarHandleDesc, {0} could not be retrieved."]*/
+					/*[MSG "K0619", "Malformed VarHandleDesc, {0} could not be retrieved."]*/
```

Comment:
This is reflected in ExternalMessages-MasterIndex.properties and should be changed there as well. It is fine with me if you want to fix this file as part of this PR. This change is not a comment change as it affects a generated file used at runtime.

Reply:
I'll get it in another pass (there are other changes to it from the `a` and `b` family)